plutus-use-cases/src/Plutus/Contracts/Governance.hs

```diff
@@ -199,7 +201,7 @@ contract params = forever $ mapError (review _GovError) endpoints where
 
     initLaw = endpoint @"new-law" $ \bsLaw -> do
         let mph = Scripts.forwardingMintingPolicyHash (typedValidator params)
-        void $ SM.runInitialise theClient (GovState (toBuiltin bsLaw) mph Nothing) mempty
+        void $ SM.runInitialise theClient (GovState (toBuiltin bsLaw) mph Nothing) (Ada.toValue Ledger.minAdaTxOut)
```

Comment:
Revert to old implementation. Should be fixed automatically when resolving comment from above.